store/src/state.rs

```diff
@@ -335,6 +340,22 @@ impl State {
         nullifiers.iter().map(|n| inner.nullifier_tree.prove(*n)).collect()
     }
 
+    /// Loads data to synchronize a client.
+    ///
+    /// The client's request contains a list of tag prefixes, this method will return the first
+    /// block with a matching tag, or the chain tip. All the other values are filter based on this
+    /// block range.
+    ///
+    /// # Arguments
+    ///
+    /// - `block_num`: The last block *know* by the client, updates start from the next block.
+    /// - `account_ids`: Include the account's hash if their _last change_ was in the result's block
+    /// range.
+    /// - `note_tag_prefixes`: Only the 16 high bits of the tags the client is interested in, result
+    /// will include notes with matching prefixes, the first block with a matching note determines
+    /// the block range.
+    /// - `nullifier_prefixes`: Only the 16 high bits of the nullifiers the client is intersted in,
+    /// results will cinlude nullifiers matching prefixes produced in the given block range.
```

Comment:
minor nit: we usually indent bullet-point lists a bit differently:
```Rust
/// - `block_num`: The last block *know* by the client, updates start from the next block.
/// - `account_ids`: Include the account's hash if their _last change_ was in the result's block
///   range.
/// - `note_tag_prefixes`: Only the 16 high bits of the tags the client is interested in, result
///   will include notes with matching prefixes, the first block with a matching note determines
///   the block range.
/// - `nullifier_prefixes`: Only the 16 high bits of the nullifiers the client is intersted in,
///   results will cinlude nullifiers matching prefixes produced in the given block range.
```

Reply:
changed